src/mixins/colors.js

```diff
@@ -81,7 +81,7 @@ export default {
   methods: {
     colorClass(args = {}) {
       let colorName = args.default || 'default';
-      Object.keys(colors).some((color) => {
+      Object.keys(colors).some(color => {
```

Comment:
qual é nossa posição com o parentesis? Em alguns lugares você colocou, em outros tirou... 🤔

Reply:
É automatizado

Reply:
Não importa. O lance do estilo de código visa, entre outras coisas, manter a consistência. Se a tal automação não consegue fazer isso ou está mal configurada ou é inútil. E colocar a culpa nela não resolve.

Reply:
Minha visão é que não é produtivo conversar sobre estilo de código. Tanta coisa mais importante e mais relevante pra ser implementar e definir onde acontece uma quebra de linha é o que vai trazer o impacto pro trabalho que fazemos. [Só conversamos sobre isso pois é facil](https://pt.wikipedia.org/wiki/Lei_da_trivialidade). A solução mais simples é deixar a automação cuidar disso e aceitar que ela não vai fazer tudo do jeito que gostamos.

Reply:
Concordo plenamente com você, desde que a solução automatizada produza um código coerente. Este não é o caso aqui e duvido muito que isso não possa ser configurado.

Reply:
Estou alterando o que os piores casos, mas vou responder os que não valer a pena muito mexer.

Reply:
Não é possível alterar isso.